BookingsApi/BookingsApi.DAL/Migrations/20220905115658_addCaseRolesToSEND.cs

```diff
@@ -0,0 +1,65 @@
+                    { 1096, HearingRoles.Intermediary, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 343, DateTime.UtcNow },
+                    { 1097, HearingRoles.Interpreter, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 343, DateTime.UtcNow },
+                    { 1098, HearingRoles.LitigantInPerson, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 343, DateTime.UtcNow },
+                    { 1099, HearingRoles.LitigationFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 343, DateTime.UtcNow },
+                    { 1100, HearingRoles.MacKenzieFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 343, DateTime.UtcNow },
+                    { 1101, HearingRoles.Representative, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 343, DateTime.UtcNow },
+                    { 1102, HearingRoles.Solicitor, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 343, DateTime.UtcNow },
+                    { 1103, HearingRoles.Witness, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 343, DateTime.UtcNow },
+                    
+                    //Respondent
+                    { 1104, HearingRoles.Appellant, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1105, HearingRoles.Expert, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1106, HearingRoles.Intermediary, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1107, HearingRoles.Interpreter, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1108, HearingRoles.LitigantInPerson, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1109, HearingRoles.LitigationFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1110, HearingRoles.MacKenzieFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+                    { 1111, HearingRoles.Representative, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 344, DateTime.UtcNow },
+                    { 1112, HearingRoles.Solicitor, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 344, DateTime.UtcNow },
+                    { 1113, HearingRoles.Witness, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 344, DateTime.UtcNow },
+            });
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+     
+            migrationBuilder.DeleteData(nameof(HearingRole), "Id", 343);
+            migrationBuilder.DeleteData(nameof(HearingRole), "Id", 344);
+            
+            for (int caseRoleId = 1081; caseRoleId <= 1100; caseRoleId++)
```

Comment:
think this should probably be 1094 - 1113. Not that we do down migrations very often